Core/clim-core/frames.lisp

```diff
@@ -683,6 +688,11 @@ documentation produced by presentations.")
 (defmethod bury-frame ((frame application-frame))
   (bury-sheet (frame-top-level-sheet frame)))
 
+(defmethod note-frame-pretty-name-changed
+    ((frame-manager frame-manager) frame new-name)
+  (when-let ((top-level-sheet (frame-top-level-sheet frame)))
+    (setf (sheet-pretty-name top-level-sheet) new-name)))
```

Comment:
ditto, note methods should not perform anything (only have default methods doing nothing to prevent errors)